nengo_gui/static/components/pointer.js

```diff
@@ -4,6 +4,10 @@
  *
  * @param {dict} args - A set of constructor arguments (see Nengo.Component)
  * @param {Nengo.SimControl} args.sim - the simulation controller
+ *
+ * Pointer function is called by python server when a user requests a plot 
```

Comment:
In the other comments you call this a constructor, rather than a function.  It should probably be called "constructor" to be consistent.